src/modules/navigator/mission_block.cpp

```diff
@@ -279,7 +279,8 @@ MissionBlock::is_mission_item_reached()
 
 			// We use the acceptance radius of the mission item if it has been set (not NAN)
 			// but only for multicopter.
-			if (_navigator->get_vstatus()->vehicle_type == vehicle_status_s::VEHICLE_TYPE_ROTARY_WING
+			if ((_navigator->get_vstatus()->vehicle_type == vehicle_status_s::VEHICLE_TYPE_ROTARY_WING
```

Comment:
The other existing vehicle types are rover and airship. I think this check here can be omitted, as these changes can be used on other vehicle types as well? Especially if you have a steering rover in mind.

Reply:
I wasn't sure about any other consequences for rover and airships (I'm new to the source code) but you are right, the acceptance radius makes sense for those vehicle types as well. Thus, this check is removed in the commit below.